doc/docs.md

```diff
@@ -6835,6 +6849,7 @@ sizeof
 spawn
 static
 struct
+thread
```

Comment:
`thread` is not a keyword, but a predefined type.
It is similar to `array`, `map`, `chan` or `int`.

Reply:
I see. Makes me think we should add Appendix III for all other reserved words.